helpers/utils.py

```diff
@@ -11,14 +11,29 @@ def format_token(token):
   else:
     return address
 
+def format_token_short(token):
+  symbol = token['symbol']
+  address = token['address']
+  name = token['name']
+  label = symbol or name
+
+  if label:
+    return label
+  else:
+    return address
```

Comment:
```suggestion
  return label if label else address
```

I know you are not a big fan of 1 liners, but this one is clear enough

Reply:
In this case I like it :) 
I usually don't like it if they do several things in the same line and you need to read all left-to-right of it to understand what is doing.